src/web/components/App.tsx

```diff
@@ -68,7 +68,7 @@ import { MapEmbedBlockComponent } from '@root/src/web/components/elements/MapEmb
 import { SpotifyBlockComponent } from '@root/src/web/components/elements/SpotifyBlockComponent';
 import { VideoFacebookBlockComponent } from '@root/src/web/components/elements/VideoFacebookBlockComponent';
 import { VineBlockComponent } from '@root/src/web/components/elements/VineBlockComponent';
-
+import { CountryCode } from '@guardian/libs/dist/esm/types/countries';
```

Comment:
Do you have to specify that this is a type?

```suggestion
import type { CountryCode } from '@guardian/libs/dist/esm/types/countries';
```